ruby/example_code/iam/scenario_create_user_assume_role.rb

```diff
@@ -0,0 +1,295 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Purpose
+#
+# Shows how to use AWS SDK for Ruby to create an IAM user, assume a role,
+# and perform AWS actions.
+#
+# 1. Create a user who has no permissions.
+# 2. Create a role that grants permission to list Amazon S3 buckets for the account.
+# 3. Add a policy to let the user assume the role.
+# 4. Assume the role and list Amazon S3 buckets using temporary credentials.
```

Comment:
Change "Amazon S3 buckets" to "S3 buckets"

Reply:
Fixed